flow/scripts/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2023 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Import command line scripts."""
```

Comment:
Can we add very basic CLI tests? Even just `flow --help`. See signac for an example of this.

Reply:
We have these tests see https://github.com/glotzerlab/signac-flow/blob/main/tests/test_shell.py.

My guess is that tests are run in the repo directory which has the file but the install doesn't due to setuptools package detection requiring an `__init__.py`.

Reply:
I see. I'll approve this.

Reply:
This is why I run tests on installed packages without any source directory. Too many issues like this can occur.